### DISCUSSION TITLE
Formula: Convert string into luxon datetime

### DISCUSSION THREAD

**Discussion by dsalzman** [RafaelGB/obsidian-db-folder#832] [Q&A]
Hi!

I have 2 tables. 1 meetings and 1 contacts. meetings have a date column. Contacts can have many meetings. I did a relationship and rollup to grab all the dates per contact into a column. 

Then I have a formula column to grab the latest meeting date. ```${new Date(row.Last_Contact.split(",").map((dates) => Date.parse(dates.trim())).sort().reverse()[0]).toLocaleDateString()}```

My question is how to convert that value back to a luxon datetime to calculate days since last contact. My current try is ```${DateTime.fromFormat(row.last_contact_date,'MM.dd.yyy').diffNow("days")}```

If anyone can help that would be great!

**Comment by RafaelGB** [comment]
hello @dsalzman ,
Which is the content of ow.last_contact_date? You can cna check the Luxon library documentation or share the content here and I will be happy to help

**Reply by dsalzman** [reply]
It's "2/18/2023" for example as type String. I think it's a casting issue.

**Reply by RafaelGB** [reply]
`DateTime.fromFormat(row.last_contact_date,'MM/dd/yyy')`

Yeah I think we have to play with the format variable
